src/Ocelot/Configuration/File/FileCacheOptions.cs

```diff
@@ -23,4 +26,6 @@ public FileCacheOptions(FileCacheOptions from)
     /// <remarks>If <see langword="null"/> then use global configuration with <see langword="false"/> by default.</remarks>
     /// <value><see langword="true"/> if content hashing is enabled; otherwise, <see langword="false"/>.</value>
     public bool? EnableContentHashing { get; set; }
+
+    public HttpStatusCodeFilter StatusCodeFilter { get; set; }
```

Comment:
```suggestion
    public int[] StatusCodes { get; set; }
```
Note that the type is not the old `HttpStatusCode[]` but rather `int[]`, an array of integers. If an Ocelot user is not a C# developer, they might not be familiar with the `HttpStatusCode` enumeration and its values!